src/app/view2/view2.component.scss

```diff
@@ -1,7 +1,18 @@
-p {
+.container {
   width: 100%;
   height: 100%;
   background-color: aliceblue;
   margin: 0;
   padding: 10px;
 }
+
+.block {
+  width: 100px;
```

Comment:
Properties should be ordered align-items, background-color, color, display, height, justify-content, margin-top, width